modules/i18n/src/translations/en-US/portals/admin-portal.ts

```diff
@@ -3004,6 +3004,13 @@ export const adminPortal: AdminPortalNS = {
             },
             title: "You are not authorized"
         },
+        sessionStorageDisabled: {
+            subtitles: {
+                0: "It looks like your browser's settings don't allow data to be stored.",
+                1: "Please enable storing data on your browser for this app to function properly."
```

Comment:
IMO this message is vague and the user might get confused. @jeradrutnam any thoughts regarding this?

Reply:
Yes, same thought I had. Better to rephrase it.

Reply:
May we can say like "**Cookies are disabled in your browser.** To use this application, you have to enable cookies in your Web browser settings. For more information on How to enable cookies, see Help section on your Web browser"

Reply:
Updated by 058471f